docs/design/test-harness-framework.md

```diff
@@ -462,7 +462,8 @@ The framework shall provide predefined flagsets that will be automatically appli
 
 * `--feature` - a regular expression that target features to run
 * `--assess` - a regular expression that targets the name of an assessment steps
-* `--labels` - a comma-separated list of key/value pairs  used to filter features by their assigned labels
+* `--labels` - a comma-separated list of key/value pairs  used to filter features by their assigned labels (key is
```

Comment:
Need to update the docs bit on `--skip-labels` as well. 

https://github.com/kubernetes-sigs/e2e-framework/blob/cd0fd6dcf181539c450224b217ac3c20f7cac185/docs/design/test-harness-framework.md?plain=1#L475